### ISSUE TITLE
Angular not rendering

### ISSUE THREAD

**Issue by 9mm** [prerender/prerender#337]
Hey there. This is a very cool service but I'm having a lot of difficulties getting it to render and I'm not sure what's wrong.

Here are some details:

I'm using Angular 1.5.8

Relevant `index.html` code I've added:

    <meta name="fragment" content="!">
    <base href="/index.html">

Relevant `app.js` code:

    $locationProvider.html5Mode(true).hashPrefix('!');

I've tried with and without `window.prerenderReady`, so I assume the problem is something else. For now I've left all if it out to see if Prerender can figure it out.

As far as our backend, we are using varnish which forwards a request to Prerender when `?prerender=1` is in the URL (a param used for testing to emulate googlebot).

Basically the problem is that Prerender seems to be 'half' working. I can tell it's doing *something* because it's removing script tags, as well as showing the `<style></style>` tag that angular automatically injects for `ng-show`, but all the other html is still using `<div>{{templating.variables}}</div>`.

Does anything about the way I'm using Prerender look wrong?

One last thing, the way were accessing the URL is like this:

    https://example.com/featured?prerender=1&_escaped_fragment_=

**Comment by thoop** [comment]
Have you tried using your own prerender server locally? I'd suggest uncommenting the logger() plugin in the server.js file and then passing the [logRequests parameter](https://github.com/prerender/prerender#logrequests).

That should show you any javascript errors on the page.

Also, you don't need to check if prerender=1 since you are using the `_escaped_fragment_` query parameter.

Post back and reopen if you don't see any javascript errors but still have issues.
